docs: add new Apify Console docs (#790)
apify/apify-docs: sources/platform/console/index.md

---
title: Apify Console
description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
sidebar_position: 1
category: platform
slug: /console
---

**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**

---

## Sign Up

To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).

![apify-console-sign-up](./images/console-signup.png)

## Log In

To log in to Apify Console, you can use one of the following methods:

* Google Single Sign-On
* GitHub Single Sign-On

> * The email and password set during account creation

Reviewer: Suggestion: Maybe I'd move this to the signup part

Author: hmm it might warrant to have them in both parts, since you can use them to sign up and log in respectively

Author: added to sing up section as well

Reviewer: To me that would be one of the "way too verbose" things. It's quite usual that if you sign up using google, you can also log in with it later.

Author: hmm I could also just remove it all together, we have screenshots there that show all the possible ways of signing up and logging in

Reviewer: Yeah, true.

Author: I think this is the way to go, I will remove it from both sections and rely for now on screenshots.

![apify-console-login](./images/console-login.png)

## Homepage Overview

![apify-console-homepage](./images/console-homepage.png)

> The Apify Console homepage provides an overview of our services. The header displays your account name and current plan level. The homepage features several sections:

Reviewer: Suggestion: I'd go with something like this instead, it's not showing our services, but the stuff relevant to the user.

```suggestion
The Apify Console homepage provides an overview of your account setup. The header displays your account name and current plan level. The homepage features several sections:
```

Also note that the homepage will probably also change quite a lot quite soon.

> * **Get Started with Apify**: This section guides you to Apify Store. Here, you can explore existing Actors or create your own. A 'Show me how' button directs you to a guided tour on the Console.

Reviewer: ```suggestion
* **Get Started with Apify**: This section guides you to Apify Store. Here, you can explore existing Actors or create your own. A 'Show me how' button directs you to a guided tour of the Console.
```

> * **Recently Viewed**: This section displays Actors you have recently accessed. New users or those who haven't viewed any Actors will see suggestions instead.

Reviewer: Note: The way we talk to the reader changes here. Above it's `you can explore`, here it is `new users or those...`

Author: Something like `If you are a new user or haven't viewed any Actor yet, you will see suggestions instead` I think that might preserve the consistency of addressing the user. Now that I think about it we might just say `If you haven't used any Actor yet, you will see suggestions instead` because it is obvious that new users would not have used any yet

* **Suggested Actors for You**: Based on your and other users' recent activities, this section recommends Actors that might interest you.

* **Actor Runs**: This section is divided into two tabs:
  * **Recent**: View your latest Actor runs.
  * **Scheduled**: Check your upcoming scheduled runs and tasks.

> Use the side menu to navigate other parts of the Apify Console easily.

Reviewer: Suggestion: This might be a good place to add a hint about keyboard shortcuts.

Author: added them in a kind of hacky way, not sure if this is good approach but seems like one that wastes least amount of space

Reviewer: Maybe I'd just list one (eg. GH) and then just refer to the shortcut to open the modal of shortcuts?

| Tab name | Description |
|:---|:---|
| [Store](/sources/platform/console/store.md)| Search for Actors that suit your web-scraping needs. |
| [Actors](/sources/platform/actors/index.mdx)| View your recent runs, Actors developed by you, rented Actors, analytics & issues, as well as:<br/><br/> &bull; Saved tasks - where you can view your saved tasks or change any settings within them.<br/> &bull; Runs - where you can check on your previous runs. |
| [Schedules](/sources/platform/schedules.md)| Schedule Actor runs & tasks to run at specified time. |
| [Storage](/sources/platform/storage/index.md)| View stored results of your runs in various data formats. |

> | [Proxy](/sources/platform/proxy/index.md)| Setup proxy for your web-scrapers. |

Reviewer: Suggestion: The page contains information about proxy usage, available proxies, and proxy credentials. I'd change the title to fit that.

Reviewer: Yeah, sorry, I mean the description.

Author: Adjusted slightly to reflect ability to view proxy usage.

Reviewer: I'd omit the "setup proxy for Actors" part, that's something that's usually done on Actor input level, not something you set globally.

---
title: Apify Console
description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
sidebar_position: 1
category: platform
slug: /console
---

**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**

---

## Sign Up

> To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).

Reviewer 2: ```suggestion
To use Apify Console, you need to create an Apify account. [Sign up here](https://console.apify.com/sign-up).
```

![apify-console-sign-up](./images/console-signup.png)

## Log In

> After signing up you need to log in using your Apify account.

Reviewer 2: ```suggestion
After signing up, you need to log in using your Apify account.
```

![apify-console-login](./images/console-login.png)

## Homepage overview

![apify-console-homepage](./images/console-homepage.png)

The Apify Console homepage provides an overview of your account setup. The header displays your account name and current plan level. The homepage features several sections:

- **Get Started with Apify**: This section guides you to Apify Store. Here, you can explore existing Actors or create your own. A 'Show me how' button directs you to a guided tour of the Console.

- **Recently Viewed**: This section displays Actors you have recently accessed. If you haven't used any Actors yet, you will see suggestions instead.

- **Suggested Actors for You**: Based on your and other users' recent activities, this section recommends Actors that might interest you.

- **Actor Runs**: This section is divided into two tabs:
  - **Recent**: View your latest Actor runs.
  - **Scheduled**: Check your upcoming scheduled runs and tasks.

> Use the side menu to navigate other parts of the Apify Console easily.

Reviewer 2: ```suggestion
Use the side menu to navigate other parts of Apify Console easily.
```
